srcalc/src/main/java/gov/va/med/srcalc/web/controller/EnterVariablesController.java

```diff
@@ -58,9 +72,12 @@ public VariableEntry constructVariableEntry(
      */
     @RequestMapping(value = "/enterVars", method = RequestMethod.GET)
     public ModelAndView presentVariableEntry(
-            final HttpSession session,
+            final HttpSession session, final HttpServletResponse response,
```

Comment:
Please update the method Javadoc to include this parameter. Since we don't actually write the content the the response, we should probably explain why we need it.